pipelines/coref/configs/cluster.cfg

```diff
@@ -0,0 +1,135 @@
+[paths]
+train = "data/train.spacy"
+dev = "data/dev.spacy"
+init_tok2vec = null
+vectors = null
+
+[system]
+gpu_allocator = "pytorch"
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["sentencizer", "transformer", "coref"]
+batch_size = 2
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.sentencizer]
+factory = "sentencizer"
+punct_chars = null
+
+[components.coref]
+factory = "experimental_coref"
+span_cluster_prefix = "coref_head_clusters"
+
+[components.coref.model]
+@architectures = "spacy-experimental.Coref.v1"
+distance_embedding_size = 20
+dropout = 0.3
+hidden_size = 1024
+depth = 1
+antecedent_limit = 100
+antecedent_batch_size = 512
+
+[components.coref.model.tok2vec]
+@architectures = "spacy-transformers.TransformerListener.v1"
+# XXX Note this freezes the transformer
+grad_factor = 0.0
+upstream = "transformer"
+
+[components.coref.model.tok2vec.pooling]
+@layers = "reduce_mean.v1"
+
+[components.transformer]
+factory = "transformer"
+
+[components.transformer.model]
+@architectures = "spacy-transformers.TransformerModel.v3"
+name = "roberta-base"
+tokenizer_config = {"use_fast": true}
+
+[components.transformer.model.get_spans]
+@span_getters = "spacy-transformers.strided_spans.v1"
+window = 400
+stride = 350
+
+[corpora]
+
+[corpora.dev]
+@readers = "spacy.Corpus.v1"
+path = ${paths.dev}
+max_length = 0
+gold_preproc = false
+limit = 0
+augmenter = null
+
+[corpora.train]
+@readers = "spacy.Corpus.v1"
+path = ${paths.train}
+max_length = 0
+gold_preproc = false
+limit = 0
+augmenter = null
+
+[training]
+dev_corpus = "corpora.dev"
+train_corpus = "corpora.train"
+seed = ${system.seed}
+gpu_allocator = ${system.gpu_allocator}
+dropout = 0.3
+accumulate_gradient = 1
+patience = 2000
+max_epochs = 100
+max_steps = 200000
+eval_frequency = 1000
+frozen_components = []
+annotating_components = ["sentencizer"]
+before_to_disk = null
+
+[training.batcher]
+@batchers = "spacy.batch_by_sequence.v1"
+size = 1
+get_length = null
+
+[training.logger]
+@loggers = "spacy.ConsoleLogger.v1"
+progress_bar = false
```

Comment:
I kinda like having it on more, because I would like to know timing plus one epoch takes a long time and it seems a bit like its hanging on the first epoch -- to me at least.

Reply:
I think we usually keep this false for demo models? I don't like it because I resize my terminal when training and the progress bar becomes a mess.